core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-properties-form/field-properties/values-property/values-property.component.ts

```diff
@@ -23,6 +23,23 @@ export class ValuesPropertyComponent {
         'com.dotcms.contenttype.model.field.ImmutableSelectField'
     ];
 
+    /**
+     * check field is required or not
+     * @readonly
+     * @type {boolean}
+     * @memberof DefaultValuePropertyComponent
+     */
+    get isRequired(): boolean {
```

Comment:
As mentioned Manuel, check if you can reuse it. Maybe can create a class and extent it in RegexCheckPropertyComponent and ValuesPropertyComponent from the same class.